### ISSUE TITLE
Moded FlySky-i6 = 10 PPM Channels on the FS-iA10B?

### ISSUE THREAD

**Issue by tshaw1** [benb0jangles/FlySky-i6-Mod-#23]
A simple question. I love the idea of the 10 Channel Mod that you have created, I am running that modification on my FlySky i6 Transmitter! And I understand that there may be hardware restrictions when trying to receive 10 PPM Channels via the FS-iA6B Receiver... However can anyone tell me that if I purchase the FS-iA10B Receiver, could I then utilize ALL 10 PPM Channels? I attempted to use Channel 7 (B Switch) with my Jiyi P2 FC to adjust flight Modes using the FC-iA6B Receiver and one cable running from Channel 1 to the RC port on my FC. It did not Recognize Channel 7.

**Comment by arun2118** [comment]
Yes, your able able to use all ten pwm out channels on ia10 receiver.

**Comment by tshaw1** [comment]
I'm interested and need to utilize the PPM Protocol. Ironically, I've found since asking the question earlier today that even the rather new i10 Transmitter will only Transmit 8 PPM channels.

**Comment by piodabro** [comment]
All FS-iAxx can provide only 8 PPM channels. Better solution is to use iBus where 10 channels are present.

**Comment by dimak771** [comment]
Hello! Can I use 10 ch with IBUS + FS-iA6B Receiver? Not PPM.
